1.8/administration/installing/cloud/aws/upgrading.md

```diff
@@ -0,0 +1,42 @@
+---
+post_title: Upgrading
+menu_order: 11
+---
+
+## Summary
+
+This document provides instructions for upgrading a DC/OS cluster from version 1.7 to 1.8 using AWS cloudformation templates. It is recommended that you familiarize yourself with the [Advanced DC/OS Installation on AWS] before proceeding.
+
+If this upgrade is performed on a supported OS with all prerequisites fulfilled, this upgrade _should_ preserve the state of running tasks on the cluster.
+
+**Important:**
+
+- The Advanced Installation method is the _only_ recommended upgrade path for DC/OS. It is recommended that you familiarize yourself with the [Advanced DC/OS Installation Guide][advanced-install] before proceeding.
```

Comment:
This will no longer be true after this doc is added. This line (and the similar one above) can be removed since the advanced install isn't relevant to this procedure. The language will have to be updated in the other upgrade docs as well.